alsoft

```diff
@@ -0,0 +1,3 @@
+# OpenAL configuration. More options at
```

Comment:
Not sure if the top-level source directory is the right place for this file. How about we put it under `data/`?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/981/commits/d9945372125dce3df29baac2d4a681379a71aa4f